specification/ai/ModelClient/client.tsp

```diff
@@ -74,10 +78,19 @@ namespace Customizations; // The actual name here doesn't matter and is here for
 @@access(AI.Model.ChatMessageImageContentItem, Access.public);
 @@access(AI.Model.ChatMessageImageUrl, Access.public);
 @@access(AI.Model.ChatMessageImageDetailLevel, Access.public);
-@@access(AI.Model.StreamingChatCompletionsUpdate, Access.public, "python,csharp");
+@@access(AI.Model.StreamingChatCompletionsUpdate,
+  Access.public,
+  "python,csharp"
+);
```

Comment:
may need to add java to this list as well (same comment for two instances below)